src/components/spinner/spinner.css

```diff
@@ -0,0 +1,41 @@
+.spinner {
+  width: 40px;
+  text-align: center;
+  display: inline-block;
+}
+
+.bounce1, .bounce2, .bounce3 {
+  width: 8px;
+  height: 8px;
+  background-color: #aaa;
+
+  border-radius: 100%;
+  display: inline-block;
+  -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+  animation: sk-bouncedelay 1.4s infinite ease-in-out both;
+}
+
+.bounce1 {
+  -webkit-animation-delay: -0.32s;
+  animation-delay: -0.32s;
+}
+.bounce2 {
+  -webkit-animation-delay: -0.16s;
+  animation-delay: -0.16s;
+}
+
+// Spinner animations
+@-webkit-keyframes sk-bouncedelay {
```

Comment:
`postcss` generates vendor prefixes, no need to add them manually. and comments with double slash (`//`) cause the next rule fail. please use `/**/`
You might like o read [this](https://css-tricks.com/snippets/css/comments-in-css/).